### PR TITLE
Blog Post: Add link to report non-.gov

### PR DESCRIPTION
**Pull request by Katherine-Osos**
## Changes proposed in this pull request:
Added link for reporting .gov domains

This update relates to #[766](https://github.com/cisagov/getgov/issues/766)

### CODE REVIEW

**Comment by Katherine-Osos** [comment]
Changes can be reviewed at [this link](https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/preview/cisagov/getgov-home/ko/link-for-non-dotgov/posts/2023-02-08-OMB-new-requirements-for-federal-executive-agencies/)

File: posts/2023-02-08-OMB-new-requirements-for-federal-executive-agencies.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ The Office of Management and Budget (OMB) issued new requirements for the regist
 
 Requirements include:
 -  **Agencies must use government domains** (i.e., .gov or .mil) for all official communications, information, and services, except for third-party services operated by non-governmental entities on non-governmental domains that are needed to effectively interact with the public. 
-- Agencies must **continue to report use of non-.gov domains** [How do they do this](#)?
+- Agencies must **continue to report use of non-.gov domains** [How do they do this](https://search.gov/about/policy/govt-urls.html){.usa-link--external}?
```
Comment (h-m-f-t) [inline posts/2023-02-08-OMB-new-requirements-for-federal-executive-agencies.md:18]: I think we should change 

> How do they do this?

to

> Report at search.gov.

**Review by h-m-f-t — CHANGES_REQUESTED** [review]
Thanks! Small change requested.

File: posts/2023-02-08-OMB-new-requirements-for-federal-executive-agencies.md (hunk 1 above)
Comment (Katherine-Osos) [reply posts/2023-02-08-OMB-new-requirements-for-federal-executive-agencies.md]: Updated!

**Review by h-m-f-t — APPROVED** [review]
Thanks!

**Pull request merged**